wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -86,19 +85,23 @@ class PlatformRepo(val walletApplication: WalletApplication) {
      * gets all the name documents for usernames starting with text
      *
      * @param text The beginning of a username to search for
-     * @param userId The current userId for which to search contacts
      * @return
      */
-    fun searchUsernames(text: String, userId: String): Resource<List<UsernameSearchResult>> {
+    suspend fun searchUsernames(text: String): Resource<List<UsernameSearchResult>> {
         return try {
+            val blockchainIdentity = blockchainIdentityDataDaoAsync.load()
+            //We don't check for nullity here because if it's null, it'll be thrown, caputred below
+            //and sent as a Resource.error
+            val userId = blockchainIdentity!!.creditFundingTxId!!.toStringBase58()
```

Comment:
I agree with this change, since I don't think we need a userId argument since it will always be a the current userId of the users blockchainIdentity.

Reply:
creditFundingTxId should be creditBurnIdentityIdentifier